lib/fathead/beautifulsoup/parse.py

```diff
@@ -78,21 +72,27 @@ def parse_first_paragraph_from_data(self, section):
         Get the first paragraph for display
         Args:
             section: A section of parsed HTML
-
         Returns:
             First paragraph in the HTML
         """
-        data = section.find('p')
+        data = section.find_all('p')
         if data:
-            return data.text.replace('\n', ' ').replace('  ', ' ')
+            arg = ''
+            for index,d in enumerate(data):
+                data[index] = d.text.replace('\n', ' ').replace('  ', ' ')
```

Comment:
@vidhan13j07 we also need to replace `\\x` with `\\\\x` here.

Some of the code snippets contain UTF-8 (like [this one](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#output-encoding)) and it's creating problems internally.

If we can escape the `\x` to `\\x` everything will work as expected 👍